pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -4535,7 +4548,8 @@ func TestResourceNames(t *testing.T) {
 			})
 			hostF := deploytest.NewPluginHostF(nil, nil, programF, loaders...)
 			p := &TestPlan{
-				Options: TestUpdateOptions{HostF: hostF},
+				// Displaytests for this are racy
```

Comment:
What's this comment for? Was this previously skipped?

Reply:
Yeah, I missed cleaning this up.  I had started excluding tests from displaytests before I realized we need to do some sorting in the display code to get consistent output.